content/resources/lessons/sound-basics/practical/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Practical
+weight: 3
+---
+
+This is a simple exercise to help you become familiar with changing the window
+size of a sound file using Audacity. This will help you to understand how
+changing the window size can alter the frequency and temporal resolution of your
+spectrograms, and affect how your spectrograms will look visually.
+
+1. Find `20220421T100000+1000_SEQP-Samford-Wet-B_644370.flac` in the 2022
+   Ecoacoustics Symposium/SERF Data folder.
+2. Open Audacity, and drag the sound file into Audacity to open it.
+![](20221206111634.png)
+1. We can view the spectrogram of our sound file by accessing the drop down arrow menu, and
+   clicking _Spectrogram_.
+![](20221206111806.png)
+1. You should now see the spectrogram view. Going back to the same drop down menu, we can
+   now select _Spectrogram Settings_, which brings up this window:
+   ![](20221206112035.png)  
+2. Take note of some important information that we should be familiar with by now:
+   1. Max Frequency: 11025 Hz. Remember that Max Frequency is half of the sampling frequency. Therefore,
+      this file was recorded with a sampling frequency of 22050 Hz. 
+   2. Window size: 2048
```

Comment:
missing context?

The default settings on a default install of audacity may not have this set to 2048 (it might, i haven't checked). The language here should represent that the window size could be any value.

Might be a good time to recap window tradeoffs.